hazelcast-client/src/test/java/com/hazelcast/client/cache/nearcache/ClientNearCacheInvalidationTest.java

```diff
@@ -0,0 +1,355 @@
+/*
```

Comment:
This client part of this test corresponds to the similar test written for cache.clear ClientCacheClearTest. Can we make both tests to be under the same package?